src/main/resources/com/google/api/codegen/java/test.snip

```diff
@@ -101,8 +101,8 @@
       client.{@test.clientMethodName}(\
           {@sampleMethodCallArgList(test.initCode.fieldSettings)});
       Assert.fail("No exception raised");
-    } catch (GrpcApiException e) {
-      Assert.assertEquals(Status.INVALID_ARGUMENT.getCode(), e.getStatusCode().getCode());
+    } catch (InvalidArgumentException e) {
+      Assert.assertEquals(Status.INVALID_ARGUMENT.getCode(), ((GrpcStatusCode) e.getStatusCode()).getCode());
```

Comment:
I don't feel that we need to actually check codes any more, since we are catching the exception that corresponds to the code. I think we should now just leave a comment like:

```
// expected exception
```

(This doesn't apply to the other type of test, which catches ExecutionException)

Reply:
Done